plugins/mongodb/src/main/java/com/navercorp/pinpoint/plugin/mongo/MongoUtil.java

```diff
@@ -86,5 +95,66 @@ public static NormalizedBson parseBson(Object[] args, boolean traceBsonBindValue
         String jsonParameterString = StringJoiner.join(jsonParameter, SEPARATOR);
         return new NormalizedBson(parsedJsonString, jsonParameterString);
     }
+
+    public static NormalizedBson parseJson(Object[] args, boolean traceJsonBindValue) {
+        StringBuilder parsedJsonBuilder = new StringBuilder();
+        StringBuilder jsonParamBuilder = new StringBuilder();
+        parsedJsonBuilder.append("{");
+        for (Object arg : args) {
+            if (arg instanceof DBObject[]) {
+                DBObject[] objects = (DBObject[]) arg;
+                for (DBObject object : objects) {
+                    extractFromDbObject(parsedJsonBuilder, jsonParamBuilder, object);
+                }
+                adjustJson(parsedJsonBuilder, jsonParamBuilder);
+            } else if (arg instanceof DBObject) {
+                extractFromDbObject(parsedJsonBuilder, jsonParamBuilder, (DBObject) arg);
+                adjustJson(parsedJsonBuilder, jsonParamBuilder);
+            } else {
+                System.err.println("Not a DBObject; it's a(n) " + arg.getClass());
+            }
+        }
+
+        String parsedJsonString = parsedJsonBuilder.toString();
+        String jsonParameterString = jsonParamBuilder.toString();
+        System.out.println("parsedJsonString - " + parsedJsonString + "; jsonParameterString - " + jsonParameterString);
```

Comment:
same here
System.err ~~ should be taken care of with something else such as log